src/db/models.py

```diff
@@ -122,6 +123,8 @@ class Sub(Model, Base):
                                                               '0=auto, 1=force display')
     display_title = fields.SmallIntField(default=0, description='Display title or not?'
                                                                 '-1=disable, 0=auto, 1=force display')
+    display_item_tags = fields.SmallIntField(default=-1, description='Display rss item tags in rss or not?'
```

Comment:
We can safely set `default=-100` here as the default value of all the customizable options of new subscriptions is explicitly set to `-100` (refer to `src.command.inner.sub`).

The reason why the DB model was not updated is that SQLite never gains the ability to alter table schema. Considering the fact that dropping a column from a table is possible since SQLite 3.35.0, manually migrating to the new default value is feasible now.

I may do this migration and bump `aerich` to `0.7.x` before or after merging this commit. You don't need to worry about DB migration. Once the PR is ready and passes my review, I will add the DB migration for the PR. In order words, I will not add DB migration for the PR now. If this makes you unable to debug, please let me know and I will make a provisional migration file.